dspace/config/crosswalks/oai/metadataFormats/oai_openaire.xsl

```diff
@@ -0,0 +1,1484 @@
+                <xsl:with-param name="value" select="$elementName"/>
+            </xsl:call-template>
+        </xsl:variable>
+        <xsl:choose>
+            <xsl:when test="$lc_title_type = 'alternativetitle' or $lc_title_type = 'alternative'">
+                <xsl:text>AlternativeTitle</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_title_type = 'subtitle'">
+                <xsl:text>Subtitle</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_title_type = 'translatedtitle'">
+                <xsl:text>TranslatedTitle</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:text>Other</xsl:text>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:template name="getContributorType">
+        <xsl:param name="elementName"/>
+        <xsl:variable name="lc_contributor_type">
+            <xsl:call-template name="lowercase">
+                <xsl:with-param name="value" select="$elementName"/>
+            </xsl:call-template>
+        </xsl:variable>
+        <xsl:choose>
+            <xsl:when test="$lc_contributor_type = 'advisor' or $lc_contributor_type = 'supervisor'">
+                <xsl:text>Supervisor</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_contributor_type = 'editor'">
+                <xsl:text>Editor</xsl:text>
+            </xsl:when>
+            <xsl:otherwise>
+                <xsl:text>Other</xsl:text>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:template name="getDateType">
+        <xsl:param name="elementName"/>
+        <xsl:variable name="lc_dc_date_type">
+            <xsl:call-template name="lowercase">
+                <xsl:with-param name="value" select="$elementName"/>
+            </xsl:call-template>
+        </xsl:variable>
+        <xsl:choose>
+            <xsl:when test="$lc_dc_date_type='available' or  $lc_dc_date_type = 'embargo'">
+                <xsl:text>Available</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='collected'">
+                <xsl:text>Collected</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='copyrighted' or $lc_dc_date_type='copyright'">
+                <xsl:text>Copyrighted</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='created'">
+                <xsl:text>Created</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='submitted'">
+                <xsl:text>Submitted</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='updated'">
+                <xsl:text>Updated</xsl:text>
+            </xsl:when>
+            <xsl:when test="$lc_dc_date_type='valid'">
+                <xsl:text>Valid</xsl:text>
+            </xsl:when>
+        </xsl:choose>
+    </xsl:template>
+
+    <xsl:template name="getRelatedIdentifierType">
```

Comment:
Please describe this template with a comment.

Reply:
Just add one description.
https://github.com/DSpace/DSpace/pull/2592/commits/7814a9a1df632f03d9d97191637e722078bba317